src/fetchapi/http.lib.js

```diff
@@ -0,0 +1,59 @@
+class SmallHTTP {
+  get(url) {
+    return new Promise((resolve, reject) => {

```

Comment:
Why do you wrap `fetch` into a `Promise`? `fetch` returns promise and there is no need to do it

Reply:
Working on it, this really has no sense